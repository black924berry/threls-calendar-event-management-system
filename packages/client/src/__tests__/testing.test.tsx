import Footer from "../components/Footer";
import renderer from "react-test-renderer";

it("testing footer component", () => {
  let tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
