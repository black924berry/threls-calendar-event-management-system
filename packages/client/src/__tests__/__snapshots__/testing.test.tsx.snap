// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing footer component 1`] = `
<div>
  Footer
</div>
`;
